src/WebJobs.Script/ScriptHostBuilderExtensions.cs

```diff
@@ -102,6 +103,12 @@ public static IHostBuilder AddScriptHost(this IHostBuilder builder, Action<Scrip
                 {
                     configBuilder.Add(new HostJsonFileConfigurationSource(applicationOptions, SystemEnvironment.Instance, loggerFactory, metricsLogger));
                 }
+                // Adding hsoting config into job host configuration
+                IConfiguration scriptHostConfiguration = applicationOptions.RootServiceProvider.GetService<IConfiguration>();
+                if (scriptHostConfiguration != null)
+                {
+                    configBuilder.AddConfiguration(scriptHostConfiguration.GetSection(ScriptConstants.FunctionsHostingConfigSectionName));
```

Comment:
What's going on here? There's already a FunctionsHostingConfigProvider registered. Why do we need changes to this for this PR?

Reply:
`FunctionsHostingConfigProvider` is registered for `ScriptHost`. `JobHost` has differerent set of `IConfiguration`. We want to copy `FunctionsHostingConfig` to `JobHost` to have access for the features in webjobs sdk.